### PR TITLE
doc: add minutes for sep 20 2016

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

**Comment by mhdawson** [comment]
PR for minutes https://github.com/nodejs/build/pull/502

File: doc/meetings/build-wg-meeting-2016-09-20.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+* Meeting video: https://www.youtube.com/watch?v=qFJh5Wagdhs
+
+## Present
+
+* Johan Bergström
+* Rod Vagg
+* Michael Dawson
+* João Reis
+* Gibson Fahnestock
+
```
Comment (jbergstroem) [inline doc/meetings/build-wg-meeting-2016-09-20.md]: Myles Borins late joiner

File: doc/meetings/build-wg-meeting-2016-09-20.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  * Once we come the current canvas of who is still involved
+    we should refresh the keys across test and release machines
+
+* Create proposal for list of officially supported platforms #488
+  * posted draft on issue, will plan to put on CTC agenda next week.
+    
+* TAP Plugin issues on Jenkins #453
+
+  * Myles: we need to update the node test runner to report tap using
+  YAMLish for the stack traces. Once that is done the tap -> junit
```
Comment (jbergstroem) [inline doc/meetings/build-wg-meeting-2016-09-20.md]: indent two more lines for consistency.

**Comment by mhdawson** [comment]
Nits address and landed as 431d9a2eab74e6f055582e5d3b42d8097c9b8ded

**Pull request closed without merging**